src/Module/Contact/Profile.php

```diff
@@ -415,6 +415,24 @@ private function getContactActions(array $contact, Entity\LocalRelationship $loc
 
 		$formSecurityToken = self::getFormSecurityToken('contact_action');
 
+		if ($localRelationship->rel & Contact::SHARING) {
```

Comment:
You can unfollow a `FRIEND` as well.

Reply:
Look at the operator, it's the binary AND, which means the following for relationships:
- None: 0 & 2 => false
- Follower: 1 & 2 => false
- Sharing: 2 & 2 => true
- Friend: 3 & 2 -> true

Reply:
It's 0:30, my primary brain is already gone to sleep.